src/renderer/app.tsx

```diff
@@ -35,6 +41,7 @@ export const App = () => {
     const { bindings } = useHotkeySettings();
     const handlePlayQueueAdd = useHandlePlayQueueAdd();
     const { clearQueue, restoreQueue } = useQueueControls();
+    const { enabled } = useRemoteSettings();
```

Comment:
Maybe a more descriptive name like `remoteEnabled`?

```suggestion
    const { enabled: remoteEnabled } = useRemoteSettings();
```